src/main/scala/it/pagopa/interop/agreementprocess/api/impl/AgreementApiServiceImpl.scala

```diff
@@ -196,6 +196,44 @@ final case class AgreementApiServiceImpl(
     }
   }
 
+  /**
+    * Code: 200, Message: Agreement updated., DataType: Agreement
+    * Code: 404, Message: Agreement not found, DataType: Problem
+    * Code: 400, Message: Invalid ID supplied, DataType: Problem
+    */
```

Comment:
I suggest to remove these auto generated comments, they are rarely maintained and then become misleading